docs/index.rst

```diff
@@ -26,6 +26,16 @@
    Docker Image </rst/installation/docker.rst>
 
 
+.. _index_spy_application:
+
+.. toctree::
+   :caption: Interacive Application
+   :maxdepth: 2
+   :numbered: 5
+   :hidden:
+
+   /rst/interactive_application/getting_started/getting_started
```

Comment:
I would skip Getting Started and add directly overview and example of usage here.
Maybe call this section Getting Started rather than Interactive Application.
(This will make more sense when we have the user manual section)